File: 장승민/CSharp/210208/210208_student/210208_student/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _210208_student
{
    class Student
    {
        private string name;        
        private int age;
        private char gender;
        private string address;

        public Student(string name, int age, char gender, string address)
        {
            this.name = name;
            this.age = age;
            this.gender = gender;
            this.address = address;
        }

        public void setName(string name)
        {
            this.name = name;
        }
        public string getName()
        {
            return name;
        }
        public void setAge(int age)
        {
            this.age = age;
        }
        public int getAge()
        {
            return age;
        }
        public void setGender(char gender)
        {
            this.gender = gender;
        }
        public char getGender()
        {
            return gender;
        }

        public void setAddress(string address)
        {
            this.address = address;
        }
        public string getAddress()
        {
            return address;
        }

        public void showStudentInfo()
        {
            Console.WriteLine("이름: "+ name);
            Console.WriteLine("나이: " + age);
            Console.WriteLine("성별: " + gender);
            Console.WriteLine("주소: " + address);
            Console.WriteLine("=======================");


        }
    }
}

File: 강병수/210113/src/com/kbs/student/Student.java
package com.kbs.student;

public class Student {

	private String name;
	private String address;
	private char gender;
	private int age;
	
	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

	public char getGender() {
		return gender;
	}

	public void setGender(char gender) {
		this.gender = gender;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

	public Student(String name, String address, char gender, int age) {
		this.name = name;
		this.address = address;
		this.gender = gender;
		this.age = age;
	}
	
	public void showStudentInfo() {
		System.out.println("이름: " + name);
		System.out.println("나이: " + age);
		System.out.println("성별: " + gender);
		System.out.println("주소: " + address);
		
	}
	
}
